esgf_utilities/esg_functions.py

```diff
@@ -942,6 +942,12 @@ def call_binary(binary_name, arguments):
     except ProcessExecutionError:
         logger.error("Could not find %s executable", binary_name)
         raise
+
+    for var in os.environ:
+        local.env[var] = os.environ[var]
+    for var in local.env:
+        logger.debug("env: %s", str(var))
+
```

Comment:
This debugging is a little bit much as it prints a large number of lines every call, but could definitely be useful. I'll leave it up to you @William-Hill . Let me know if we should keep it or lose it.

Reply:
I think keeping is fine. We can just change the logging level to not include debug when we go to production